src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelector('ul');
+const employees = [...listOfEmployees.children];
+
+function sortList(list) {
+  function employeeSalary(employee) {
+    return employee.dataset.salary
+      .slice(1)
+      .replace(/,/gi, '');
+  };
```

Comment:
Let's immediately return a number from this function instead of doing it later.
You can use Number() to convert string to number.